packages/expo-random/src/__tests__/Random-test.ts

```diff
@@ -0,0 +1,15 @@
+import { mockProperty, unmockProperty } from 'jest-expo';
+
+import ExpoRandom from '../ExpoRandom';
+import * as Random from '../Random';
+
+it(`invokes native method correctly`, async () => {
+  mockProperty(ExpoRandom, 'getRandomIntegerAsync', jest.fn(async () => ''));
+
+  const value = await Random.getRandomIntegerAsync(52);
```

Comment:
```suggestion
  const value = await Random.getRandomIntegerAsync(0);
```

(We get back an empty string from the mocked native layer = 0 bytes)